PaddleHQ/transaction-search
# TOOLS-315 Add optional email filter

### Ticket(s):
Link to ticket: https://paddle.atlassian.net/browse/TOOLS-315

### Checklist

### What has been done:
`email` has been added as an optional parameter. Note that the previous parameter configuration is unchanged: date range + (amount or last 4 digits) is still required.

### How to test:
1. Create 2 transactions with different emails
1. Search for your transaction with all 5 parameters
1. Confirm that only 1 transaction is returned
1. Search for your transaction with email and 1 other parameter
1. Confirm that only 1 transaction is returned
